portals/developer-portal/src/styles/manage-keys.css

```
@@ -1,3 +1,951 @@
+
+.keys-section .mk-km-badge-sandbox {
+    border-color: #5c9fcf;
+    color: #5c9fcf;
+    background: color-mix(in srgb, #5c9fcf 8%, transparent);
+}
+
+.keys-section .mk-km-badge-dot {
+    width: 7px;
+    height: 7px;
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)